recipe/meta.yaml

```diff
@@ -2,40 +2,86 @@
-  sha256: 477a885b260c8cf8c794e0c9af0769303706f11e51367dad48e5b402ad0a9493                               # [linux64]
-  url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ aarch64_version }}-arm64-linux-binary.tar.bz2     # [aarch64]
-  sha256: ddac6499f36c18ecbce9822a53ef3914c0def5276a457446a456c62999b16d36                               # [aarch64]
-  url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ osx_version }}-x86-64-darwin-binary.tar.bz2       # [osx and x86_64]
-  sha256: 6f504b9282c83842c0601600f5e0663209b53b133125ec3334ff7a0bd0037ca6                               # [osx and x86_64]
-  url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ osx_arm64_version }}-arm64-darwin-binary.tar.bz2  # [osx and arm64]
-  sha256: 1d01fac2d9748f769c9246a0a11a2c011d7843337f8f06ca144f5a500e10c117                               # [osx and arm64]
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ linux_version }}-x86-64-linux-binary.tar.bz2      # [linux64]
+    sha256: 477a885b260c8cf8c794e0c9af0769303706f11e51367dad48e5b402ad0a9493                               # [linux64]
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ aarch64_version }}-arm64-linux-binary.tar.bz2     # [aarch64]
+    sha256: ddac6499f36c18ecbce9822a53ef3914c0def5276a457446a456c62999b16d36                               # [aarch64]
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ osx_version }}-x86-64-darwin-binary.tar.bz2       # [osx and x86_64]
+    sha256: 6f504b9282c83842c0601600f5e0663209b53b133125ec3334ff7a0bd0037ca6                               # [osx and x86_64]
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ osx_arm64_version }}-arm64-darwin-binary.tar.bz2  # [osx and arm64]
+    sha256: 1d01fac2d9748f769c9246a0a11a2c011d7843337f8f06ca144f5a500e10c117                               # [osx and arm64]
+
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ win_64_version }}-x86-64-windows-binary.msi       # [win and x86_64]
+    sha256: 2cf132de1688b6e7d108128525446ae3de0fb7b98e3af016a1278593878be5a2                               # [win and x86_64]
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ win_64_version }}-source.tar.bz2                  # [win and x86_64]
+    sha256: 8a932627b3f1d8e9618f1cdc225edcb002456804697e2c87d140683764a106d5                               # [win and x86_64]
+    folder: sbcl-source                                                                                    # [win and x86_64]
+    patches:
+      - patches/0001-win-use-env-xargs.sh.patch  # [win]
+      - patches/0002-win-update-flags-zstd.patch  # [win]
 
 build:
-  number: 2
-  skip: true  # [win]
+  number: 3
+  script_env:
+    - MSI_FILE=sbcl-{{ win_64_version }}-x86-64-windows-binary.msi  # [win and x86_64]
 
 requirements:
   build:
-    - {{ stdlib("c") }}
-    - make  # [unix]
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - make
+    - zstd
```

Comment:
This should probably only be in host. If not, add a comment.
```suggestion
```

Reply:
SBCL needs zstd in order to save compressed core (in the INSTALL). I opt to add a comment. Note that we could remove this feature instead